Vision/README.md

```diff
@@ -103,7 +103,8 @@ We must
 	minimizing single points of failure 
 	and single points of control.
 * Remain focused on interoperability 
-	and ensure our work is supported by open test suites.
+	by ensuring adequate implementation experience
+	and ensuring our work is supported by open test suites.
```

Comment:
```suggestion
	 ensuring our work is supported by open test suites and implementation experience.
```

Reply:
I agree with the points Tantek made, this is an alternate wording suggestion.

Reply:
I think it's very important that we both retain and center "interoperability" in this point. Perhaps I am misunderstanding your suggestion Chaals — could you write out the full sentence with the edits you are suggesting?

Reply:
it collapses the two lines you see from @cwilso's PR. The intro line 105 remains unchanged, so it would become

> Remain focused on interoperability ensuring our work is supported by open test suites and implementation experience.

(A comma after interoperability, and perhaps "reliable" before as you suggested, might be further improvements...)

Reply:
Changed to Tantek's suggestion, because I was getting lost.  I don't like swapping the order (I think implementation experience should come before test suites).